server/src/main/scala/org/dbpedia/extraction/server/stats/MappingStatsHolder.scala

```diff
@@ -51,7 +51,7 @@ object MappingStatsHolder {
         }
       }
       
-      val redirects = wikiStats.redirects.filterKeys(title => templateMappings.contains(title)).map(_.swap)
+      val redirects = wikiStats.redirects.filterKeys(title => templateMappings.contains(title.substring(templateNamespace.length))).map(_.swap)
```

Comment:
actually both :) it's the part of the PR about fixing existing code.
I just realized I have explained the whereabouts in the issue about redirected templates but not in the PR itself... updating with that.